sklearn/impute/_iterative.py

```diff
@@ -508,6 +513,7 @@ def _initial_imputation(self, X):
         _check_inputs_dtype(X, self.missing_values)
 
         mask_missing_values = _get_mask(X, self.missing_values)
+        X_missing_mask = mask_missing_values
```

Comment:
There is something to clarify here since both `mask_missing_values` and `X_missing_mask` have the same shape. We should probably update the documentation.

I think that it would be best to explicitely make a copy because this is not obvious why we are affecting 2 variables to the same object.

Reply:
Will both of them have the same shape? As we are extracting only `valid_mask` from `mask_missing_values`. I'm not sure about it.